js/components/date.js

```diff
@@ -70,5 +64,15 @@ Fliplet.FormBuilder.field('date', {
       this.empty = false;
     }
     $vm.$v.$reset();
+  },
+  watch: {
+    value: function(val) {
+      if (Fliplet.Env.get('platform') === 'web') {
```

Comment:
`if (Fliplet.Env.is('web')) {}`

Reply:
@squallstar should I add this? Because you have approved it already.